templates/patterns/featured_item/featured_item.ui_patterns.yml

```diff
@@ -5,17 +5,15 @@ featured_item:
     default:
       label: "Default"
       description: "Default type of the featured item."
-    extended:
-      label: "Extended"
-      description: "External type of the featured item."
+    tile:
+      label: "Tile"
+      description: "Simple type of the featured item."
   fields:
-    image:
-      type: "array"
-      label: "Image"
-      description: "Image of the featured item."
-      preview:
-        src: "https://placeimg.com/1000/500/arch"
-        alt: "Alternative text for featured item image"
+    description:
+      type: "text"
+      label: "Description"
+      description: "Description of the featured item."
+      preview: "Transparently designing and evaluating evidence-based EU legislation, backed by citizens views."
```

Comment:
This field did not exists in our pattern and shouldn't be added (unless requested by business). Let's leave it out for now.

Reply:
Let's keep this instead, if empty it will not be displayed.